config/software/gcc.rb

```diff
@@ -22,9 +23,13 @@
 dependency "mpc"
 dependency "libiconv"
 
+# version_list: url=https://ftp.gnu.org/gnu/gcc/ filter=*.tar.gz
 version("4.9.2") { source sha256: "3e573826ec8b0d62d47821408fbc58721cd020df3e594cd492508de487a43b5e" }
-version("4.9.3") { source sha256: "e6c63b40877bc756cc7cfe6ca98013eb15f02ec6c8c2cf68e24533ad1203aaba" }
```

Comment:
For the time being let's not remove any versions. We can be sure that community folks aren't using them.

Reply:
okay, will make the changes